src/pages/LoginScreen.js

```diff
@@ -58,7 +60,13 @@ export class LoginScreen extends React.Component {
               secureTextEntry={true}
               placeholder="Password"
               onChangeText={(password) => this.setState({password})}
-            />
+            /> */}
+            <AppTextInput
```

Comment:
The **Enter** key when entering user name should be **Next** which will focus the password TextInput on press.
**_The above function was working before these changes._**